### DISCUSSION TITLE
Edit draft template of security advisory

### DISCUSSION THREAD

**Discussion by c0wjay** [community/community#44436] [New to GitHub]
Hello. I'd like to use github's security advisory feature for bug bounty.
However, when I open a new draft of security advisory,
a fixed template for the draft (below) is created, and I can't find how to make custom template.
Is there any way to modify the draft template? My expectation is sth like creating `PULL_REQUEST_TEMPLATE.md` or `ISSUE_TEMPLATE.md` under the .github folder while we do similar things in PR/Issue templates.

```
### Impact
_What kind of vulnerability is it? Who is impacted?_
a
### Patches
_Has the problem been patched? What versions should users upgrade to?_

### Workarounds
_Is there a way for users to fix or remediate the vulnerability without upgrading?_

### References
_Are there any links users can visit to find out more?_
```

**Comment by chris-c-thomas** [comment]
So I scoured through every area of the docs I could find that might include this. Would definitely follow that way of creating templates though since there are types of options that a `.md` file wouldn't be able to handle.

[PR's query parameters syntax](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/using-query-parameters-to-create-a-pull-request)
[Issues `.yml` syntax](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms)
[Discussions `.yml` syntax](https://docs.github.com/en/discussions/managing-discussions-for-your-community/syntax-for-discussion-category-forms)
[GitHub form schema `.yml` syntax](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema)

So something like `.github/SECURITY_ADVISORIES/config.yml` makes sense. Searching through https://github.com/github/docs as best I could didn't turn anything else up regarding Security Advisories though.
